frontend/src/routes/+page.svelte

```diff
@@ -1,109 +1,137 @@
+							<option value="8">8</option>
+							<option value="9">9</option>
+							<option value="10">10</option>
+							<option value="11">11</option>
+							<option value="12">12</option>
+						</select>
+						<span class="mx-1 mt-2 font-bold text-gray-950">:</span>
+						<select class="w-20 py-2 px-4 rounded-full border border-white bg-white focus:outline-none focus:ring focus:border-blue-300 text-gray-950">
+							<option value="" disabled selected class="text-secondary-400">min</option>
+							<option value="00">00</option>
+						  	<option value="15">15</option>
+						  	<option value="30">30</option>
+						  	<option value="45">45</option>
+						</select>
+					</div>
+					<span class="mx-1 mt-2 font-bold text-gray-950">-</span>
+					<div class="flex bg-white rounded-full">
+						<select class="w-20 py-2 px-4 rounded-full border border-white bg-white focus:outline-none focus:ring focus:border-blue-300 text-gray-950">
+							<option value="" disabled selected class="text-secondary-400">hr</option>
+							<option value="1">1</option>
+							<option value="2">2</option>
+							<option value="3">3</option>
+							<option value="4">4</option>
+							<option value="5">5</option>
+							<option value="6">6</option>
+							<option value="7">7</option>
+							<option value="8">8</option>
+							<option value="9">9</option>
+							<option value="10">10</option>
+							<option value="11">11</option>
+							<option value="12">12</option>
+						</select>
+						<span class="mx-1 mt-2 font-bold text-gray-950">:</span>
+						<select class="w-20 py-2 px-4 rounded-full border border-white bg-white focus:outline-none focus:ring focus:border-blue-300 text-gray-950">
+							<option value="" disabled selected class="text-secondary-400">min</option>
+							<option value="00">00</option>
+							<option value="15">15</option>
+							<option value="30">30</option>
+							<option value="45">45</option>
+						</select>
+					</div>
+					
+					<button type="submit" value="searchTime" class="py-1 px-2 rounded-full border border-gray-300 bg-white focus:outline-none focus:ring focus:border-blue-300 text-gray-950">
+						<img src="img/mag.svg" alt="Submit" style="width: 20px; height: 20px;">
+					</button>
+
+				</div>
 			</div>
-		</div>
+		</form>
 	</div>
-</div>
+	<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```

Comment:
Using `br` for spacing is bad practice

Reply:
My bad; I forgot to remove them before sending the pull request. I just threw them in real quick to see what the page would look like while scrolling down